std/traits.d

```diff
@@ -421,16 +421,40 @@ template ImmutableOf(T)
     static assert(is(  ImmutableOf!int ==    immutable int));
 }
 
-/// Get qualifier template from the given type T
+/**
+ * Gives a template that can be used to apply the same
+ * attributes that are on the given type `T`. E.g. passing
+ * `inout shared int` will return `SharedInoutOf`.
+ *
+ * Params:
+ *     T = the type to check qualifiers from
+ * Returns:
+ *     The qualifier template from the given type `T`
+ */
 template QualifierOf(T)
 {
-         static if (is(T == shared(const U), U)) alias QualifierOf = SharedConstOf;
-    else static if (is(T ==        const U , U)) alias QualifierOf = ConstOf;
-    else static if (is(T == shared(inout U), U)) alias QualifierOf = SharedInoutOf;
-    else static if (is(T ==        inout U , U)) alias QualifierOf = InoutOf;
-    else static if (is(T ==    immutable U , U)) alias QualifierOf = ImmutableOf;
-    else static if (is(T ==       shared U , U)) alias QualifierOf = SharedOf;
-    else                                         alias QualifierOf = MutableOf;
```

Comment:
I think this was purposefully formatted like this to allow a brief overview on all covered cases.

Reply:
I can revert it but the reason I changed it is it doesn't technically conform to the style guide.

Reply:
I don't feel strongly either way, maybe we wait one or two days, s.t. someone who feels strongly can raise their voice?